jobs/JGDAS_ENKF_DIAG

```diff
@@ -91,28 +91,28 @@ export COMOUT="${ROTDIR}/enkf${CDUMP}.${PDY}/${cyc}"
 
 
 export ATMGES_ENSMEAN="${COMIN_GES_ENS}/${GPREFIX}atmf006${GSUFFIX}"
-if [ ! -f ${ATMGES_ENSMEAN} ]; then
+if [[ ! -f ${ATMGES_ENSMEAN} ]]; then
     echo "FATAL ERROR: FILE MISSING: ATMGES_ENSMEAN = ${ATMGES_ENSMEAN}"
     exit 1
 fi
 
 
 # Link observational data
 export PREPQC="${COMIN_OBS}/${OPREFIX}prepbufr"
-if [ ! -f ${PREPQC} ]; then
+if [[ ! -f ${PREPQC} ]]; then
     echo "WARNING: Global PREPBUFR FILE ${PREPQC} MISSING"
 fi
 export TCVITL="${COMIN_ANL}/${OPREFIX}syndata.tcvitals.tm00"
 if [[ ${DONST} = "YES" ]]; then
    if [[ ${MAKE_NSSTBUFR} == "YES" ]]; then
-      export NSSTBF="${COMOUT}/${OPREFIX}nsstbufr"
+      export NSSTBF="${COMIN_OBS}/${OPREFIX}nsstbufr"
```

Comment:
Looks like these two branches are identical now. Can we remove the `if`?

Reply:
Looks like the NSSTBUFR should be allowed to be made internal or external. It may be better to keep this if here for now to allow developer assign para location as need.